### ISSUE TITLE
Do not wait for Strapi Analytics to respond

### ISSUE THREAD

**Issue by derrickmehaffy** [strapi/strapi#18441]
### Required System information

- Node.js version: v20
- NPM version: v8
- Strapi version: 4.14.4
- Database: Any / N/A
- Operating system: Linux Mint 21.2
- Is your project Javascript or Typescript: Javascript

### Describe the bug

We we should not be awaiting a response from the analytics server as it causes slow loading when our analytics server is under a heavy load

### Steps to reproduce the behavior

Reproduction is difficult as it depends on if the analytics accepter is under a heavy amount of load but this appears to be a regression as I know we have removed await from analytics requests in the past

### Expected behavior

We should not wait for the analytics to respond as it provides nothing the app needs and all analytics requests should be wrapped in a try/catch where we don't do anything with an error.

### Screenshots

N/A unless @ComfortablyCoding can provide some screenshots when he sees it or others

### Code snippets

Good:

https://github.com/strapi/strapi/blob/8632eeaf18063ea063b9acc1b2771460123d84a7/packages/core/strapi/src/utils/postinstall.ts#L15

https://github.com/strapi/strapi/blob/8632eeaf18063ea063b9acc1b2771460123d84a7/packages/generators/app/src/utils/usage.ts#L88

Bad:
https://github.com/strapi/strapi/blob/8632eeaf18063ea063b9acc1b2771460123d84a7/packages/core/strapi/src/services/metrics/sender.ts#L92

https://github.com/strapi/strapi/blob/8632eeaf18063ea063b9acc1b2771460123d84a7/packages/core/admin/admin/src/pages/App/index.js#L131

https://github.com/strapi/strapi/blob/8632eeaf18063ea063b9acc1b2771460123d84a7/packages/core/strapi/src/commands/actions/telemetry/disable/action.ts#L35

https://github.com/strapi/strapi/blob/8632eeaf18063ea063b9acc1b2771460123d84a7/packages/core/helper-plugin/src/features/Tracking.tsx#L286

https://github.com/strapi/strapi/blob/8632eeaf18063ea063b9acc1b2771460123d84a7/packages/core/strapi/src/commands/actions/telemetry/enable/action.ts#L72

### Additional context

Side point we hard code the analytics URL sometimes and other times we use a var, should probably clean that up.

**Comment by ComfortablyCoding** [comment]
The result of these awaits (at least the one in the admin) is that the application hangs while it waits for the request.

I was experiencing 3-5s of wait time for analytics response causing page load to take at least that long on first load.

I will post any screenshots here if I experience it again

**Comment by joshuaellis** [comment]
I agree with the admin, I'm not sure why we should wait to potentially ping the analytics server. But the helper plugin is imo correct, because it's up to the user of the callback to decide if they want to await it surely? Most 'trackUsage' calls are not awaited, I think... 🤔

**Comment by derrickmehaffy** [comment]
I'll need to look after I get back from vacation but that useTracking is all over the admin:

https://github.com/search?q=repo%3Astrapi%2Fstrapi%20useTracking&type=code

**Comment by joshuaellis** [comment]
> I'll need to look after I get back from vacation but that useTracking is all over the admin:
> 
> 
> 
> https://github.com/search?q=repo%3Astrapi%2Fstrapi%20useTracking&type=code

It is, yes, but if we're not awaiting it then we're not hanging anything. Also event handlers aren't typically sync behaviour.

**Comment by ComfortablyCoding** [comment]
> > I'll need to look after I get back from vacation but that useTracking is all over the admin:
> > https://github.com/search?q=repo%3Astrapi%2Fstrapi%20useTracking&type=code
> 
> It is, yes, but if we're not awaiting it then we're not hanging anything. Also event handlers aren't typically sync behaviour.
> 
> But to iterate the admin should be fixed for sure, community prs welcome 🌞

trackUsage which is in the useTracking hook does have an await 
https://github.com/strapi/strapi/blob/master/packages/core/helper-plugin/src/features/Tracking.tsx#L279C9-L279C19

**Comment by derrickmehaffy** [comment]
AFAIK we shouldn't really care what the response is since I don't think the tracker actually returns anything useful

**Comment by innerdvations** [comment]
Yes, the rule is to never await analytics. I have one specific case in the CLI I added where it is necessary to prevent the process from ending early, but even that I consider a bug in need of a workaround. All the other instances should be fixed.
